src/client/components/admin/UserTable.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import './UserTable.scss';
+
+interface UserTableProps {}
+
+class UserTable extends React.Component<{}, { userData: Array<React.ReactElement<any>> }> {
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            userData: [],
+        };
+    }
+
+    public async componentDidMount(): Promise<void> {
+        const result = await fetch('/graphql', {
```

Comment:
1. This will never work post-merge because you're not passing credentials
2. I'm in favor of not using redux state for this since this data expires every time you visit this page and it only appears on this page, but you should do more to separate the presentation and logical side of this component. Fetch this data in a container and render it in a presentational SFC.